nixos/modules/services/monitoring/prometheus/exporters/ipsec.nix

```
@@ -0,0 +1,21 @@
+{ config, lib, pkgs, options }:
+
+with lib;
```

```suggestion
with lib;

```
this is an antipattern and this file only uses one thing once `concatStringsSep` from lib, so there's really no need for this `with`